File: src/test/java/com/lzhs/luckin_coffee/LuckinCoffeeApplicationTests.java
package com.lzhs.luckin_coffee;

import org.junit.jupiter.api.Test;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class LuckinCoffeeApplicationTests {
    private final Logger logger= LoggerFactory.getLogger(LuckinCoffeeApplicationTests.class);

    @Test
    void contextLoads() {
        logger.debug("debug....");
        logger.info("info....");
        logger.error("error....");

    }

}

File: src/test/kotlin/com/luckin/LuckinCoffeeApplicationTests.kt
package com.luckin

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class LuckinCoffeeApplicationTests {

	@Test
	fun contextLoads() {
		println("""
			这个是第一行
					第二行
结束！
		""".trimIndent())
	}

}
